controllers/droneController.js

```diff
@@ -0,0 +1,78 @@
+const Drone = require("./../models/Drone.model");
+
+//listar drones
+exports.getAllDrones = async (req, res) => {
+  const allDrones = await Drone.find({});
+  // console.log(allDrones);
+  res.render("drones/list", {
+    data: allDrones,
+  });
+};
+
+//crear drone
+//1.vista
+exports.viewCreateDrone = async (req, res) => {
+  res.render("drones/create-form");
+};
+
+//2.datos
+exports.createDrone = async (req, res) => {
+  //obtener datos del formulario
+  console.log(req.body);
+
```

Comment:
Con esto nos podemos ahorrar muchas lineas lo vimos en M1 req.body es un objeto destructuramos aca y las podemos sacar todas en una sola linea
```suggestion
  const {name,propellers, maxSpeed, image} = req.body;
```